### PR TITLE
Update authorization to edit widget

### PR DESCRIPTION
**Pull request by Matteo-Peronnet**
## Type
Feature 

## Purpose

Victoire basic user can't edit the original widget from a child page.

## BC Break
NO

### CODE REVIEW

File: Tests/App/src/Acme/AppBundle/DataFixtures/Seeds/ORM/User/user.yml [hunk 1]
```diff
@@ -22,3 +22,16 @@ Victoire\Bundle\UserBundle\Entity\User:
...
+        email: luke@victoire.io
+        plainPassword: test
+        firstname: Luke
+        lastname: Skywalker
+        enabled: true
+        locale: en
+        createdAt: <dateTime('now')>
+        updatedAt: <dateTime('now')>
+        roles:
+            ROLE_ADMIN: ROLE_ADMIN
```
Comment (alexislefebvre) [inline Tests/App/src/Acme/AppBundle/DataFixtures/Seeds/ORM/User/user.yml]: Only values will be stored in the database, so I think that we can simplify this line:

```suggestion
            - ROLE_ADMIN
```

File: Tests/App/src/Acme/AppBundle/DataFixtures/Seeds/ORM/User/user.yml [hunk 2]
```diff
@@ -22,3 +22,16 @@ Victoire\Bundle\UserBundle\Entity\User:
...
+            ROLE_VICTOIRE_DEVELOPER: ROLE_VICTOIRE
```
Comment (alexislefebvre) [inline Tests/App/src/Acme/AppBundle/DataFixtures/Seeds/ORM/User/user.yml]: Only values will be stored in the database, so I think that we can simplify this line:

```suggestion
            - ROLE_VICTOIRE
```

File: Tests/App/src/Acme/AppBundle/DataFixtures/Seeds/ORM/User/user.yml (hunk 1 above)
Comment (alexislefebvre) [inline Tests/App/src/Acme/AppBundle/DataFixtures/Seeds/ORM/User/user.yml]: The default user should not be an admin.

```suggestion
```

File: Tests/Features/Context/VictoireContext.php [hunk 3]
```diff
@@ -132,6 +132,17 @@ public function iLoginAsVisitor()
...
     }
 
+    /**
+     * @Given I login as Victoire default user
+     */
+    public function iLoginAsVictoireDefaultUser()
+    {
+        $this->getSession()->getDriver()->stop();
+        $baseUrl = $this->minkContext->getMinkParameter('base_url');
+        $url = str_replace('anakin@victoire.io:test', 'luke@victoire.io:test', $baseUrl);
```
Comment (lenybernard) [inline Tests/Features/Context/VictoireContext.php]: I'm not sure about this... I would rather to pass the user email to the function parameters
```diff
    /**
-     * @Given I login as Victoire default user
+     * @Given I change default user
+     * @Given I use :email as default user
     */
-    public function iLoginAsDefaultUser()
+    public function iChangeDefaultLoginUser($email = 'luke@victoire.io')
    {
        $this->getSession()->getDriver()->stop();
        $baseUrl = $this->minkContext->getMinkParameter('base_url');
-       $url = str_replace('anakin@victoire.io:test', 'luke@victoire.io:test', $baseUrl);
+       $url = str_replace('anakin@victoire.io:test', $email.':test', $baseUrl);
```

Comment (Matteo-Peronnet) [reply Tests/Features/Context/VictoireContext.php]: Yes, i'v made some change, now i use a generic function to change default login user

**Pull request merged**